pynopegl-utils/pynopegl_utils/viewer/__init__.py

```diff
@@ -39,6 +39,13 @@
 import pynopegl as ngl
 
 
+def _uri_to_path(uri):
+    path = QUrl(uri).path()  # Handle file://
```

Comment:
Don't want to keep the original comment verbatim? :(

Reply:
Done. My intent was to drop the comment entirely as it is explicit that we are dealing with uri here.